Firmware/mmu2.cpp

```diff
@@ -83,15 +83,13 @@ static constexpr E_Step ramming_sequence[] PROGMEM = {
     { -35.0F,   2000.0F / 60.F},
 };
 
-static constexpr E_Step load_to_nozzle_sequence[] PROGMEM = { 
-    { 10.0F,  810.0F / 60.F}, // feed rate = 13.5mm/s - Load fast until filament reach end of nozzle
-    { 25.0F,  198.0F / 60.F}, // feed rate = 3.3mm/s  - Load slower once filament is out of the nozzle
+static constexpr E_Step load_to_nozzle_sequence[] PROGMEM = {
+    { MMU2_EXTRUDER_PTFE_LENGTH,       810.0F / 60.F}, // feed rate = 13.5mm/s - Load fast while not at heatbreak
+    { MMU2_EXTRUDER_HEATBREAK_LENGTH,  198.0F / 60.F}, // feed rate = 3.3mm/s  - Load slower once filament reaches heatbreak
```

Comment:
Is this correct? Seems like the magnitude of the two parameters has changed here, i.e. we've gone from a shorter value first to a longer value in the first entry...

Reply:
To be honest I’m not sure if this is correct. It is my understand that “Load to nozzle” should leave the filament at the very end of the nozzle (just far enough to not extrude out of the nozzle). But if this is incorrect, then I need some definition for the lengths. 

The Prusa testing team reported that Load to Nozzle was extruding too little. So I am assuming here thats because 35mm extruder sequence is too short. 

In the loading test we added last week, we use the full distance as I’m proposing in this PR. Maybe thats not correct either.